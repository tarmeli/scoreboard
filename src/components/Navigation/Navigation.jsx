import React from "react";
import Nav from "../Element/Nav/Nav";
import NavigationItemList from "./NavigationItemList/NavigationItemList";

const Navigation = () => (
    <Nav>
        <NavigationItemList />
    </Nav>
);

export default Navigation;
