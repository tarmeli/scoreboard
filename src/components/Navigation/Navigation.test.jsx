import React from "react";
import { mount } from "enzyme";

import Navigation from "./Navigation";
import { MemoryRouter } from "react-router-dom";

describe("Navigation", () => {
    let component;

    beforeEach(() => {
        component = mount(
            <MemoryRouter>
                <Navigation />
            </MemoryRouter>
        );
    });

    it("renders", () => {
        expect(component.render()).toMatchSnapshot();
    });
});
