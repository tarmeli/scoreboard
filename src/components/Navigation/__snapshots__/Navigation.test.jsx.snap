// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders 1`] = `
<nav>
  <ul>
    <li>
      <a
        alt="Home page"
        href="/"
      >
        Home
      </a>
    </li>
    <li>
      <a
        alt="List of games"
        href="/games"
      >
        Games
      </a>
    </li>
    <li>
      <a
        alt="User's profile"
        href="/profile"
      >
        Profile
      </a>
    </li>
  </ul>
</nav>
`;
